pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3,7 +3,8 @@
     "Microsoft.CSharp": {
       "StableVersions": [
         "4.0.1",
-        "4.0.0"
+        "4.0.0",
```

Comment:
I'm assuming you used some sort of tool to update the index and have verified that all the packages are stable now from a clean build.